src/BlockHorizons/BlockPets/commands/PetCommand.php

```diff
@@ -43,7 +43,7 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 						$sender->sendMessage(
 							TextFormat::GREEN . "/spawnpet <petType> [petName|select] [size] [isBaby] [player]: " . TextFormat::YELLOW . "Spawns a new pet with the given data." . PHP_EOL .
 							TextFormat::GREEN . "/removepet <name> [player]: " . TextFormat::YELLOW . "Removes the first pet with the given name, and checks for a player if given." . PHP_EOL .
-							TextFormat::GREEN . "/togglepet [all/name]: " . TextFormat::YELLOW . "Toggles your pets on/off, depending on the current state." . PHP_EOL .
+							TextFormat::GREEN . "/togglepet <pet name> [player]: " . TextFormat::YELLOW . "Toggles a pet on/off, depending on the current state." . PHP_EOL .
```

Comment:
Do note that people can use /togglepet all to toggle all pets, you might just want to keep that there.

Reply:
Duly Noted and corrected.